azure-tools-common/src/main/java/com/microsoft/azure/common/utils/AppServiceUtils.java

```diff
@@ -124,4 +128,33 @@ public static DockerImageType getDockerImageType(final String imageName, final b
             return hasCredential ? DockerImageType.PRIVATE_DOCKER_HUB : DockerImageType.PUBLIC_DOCKER_HUB;
         }
     }
+
+    public static String getSubscriptionId(String resourceId) {
+        return getSegment(resourceId, "subscriptions");
+    }
+
+    public static RuntimeStack parseRuntimeStack(String linuxFxVersion) {
+        if (StringUtils.isEmpty(linuxFxVersion)) {
+            return null;
+        }
+        final String[] segments = linuxFxVersion.split(Pattern.quote("|"));
+        if (segments.length != 2) {
+            return null;
+        }
+        return new RuntimeStack(segments[0], segments[1]);
+    }
+
+    public static boolean isDockerWebapp(WebApp webapp) {
+        final String linuxFxVersion = webapp.linuxFxVersion();
+        return StringUtils.containsIgnoreCase(linuxFxVersion, "DOCKER|");
+    }
+
+    private static String getSegment(String id, String segment) {
```

Comment:
There has been a same method in utils https://github.com/microsoft/azure-maven-plugins/blob/develop/azure-tools-common/src/main/java/com/microsoft/azure/common/Utils.java#L79

Reply:
Done